opensquirrel/decomposer/__init__.py

```diff
@@ -1,11 +1,9 @@
-from opensquirrel.decomposer.aba_decomposer import (
-    XYXDecomposer,
-    XZXDecomposer,
-    YXYDecomposer,
-    YZYDecomposer,
-    ZXZDecomposer,
-    ZYZDecomposer,
-)
+from opensquirrel.decomposer.aba_decomposer import (XYXDecomposer,
```

Comment:
I would merge develop into this branch because Stan and I fixed many linting issues after upgrading and strictifying Ruff

Reply:
Will do!